addons/mail/views/discuss_channel_views.xml

```diff
@@ -30,6 +30,23 @@
             </field>
         </record>
 
+        <record id="mail.discuss_channel_view_list" model="ir.ui.view">
+            <field name="name">discuss.channel.list</field>
+            <field name="model">discuss.channel</field>
+            <field name="priority" eval="10"/>
+            <field name="arch" type="xml">
+                <list string="Channels">
+                    <field name="name"/>
+                    <field name="description"/>
+                    <field name="is_member" column_invisible="True"/>
+                    <field name="group_ids" column_invisible="True"/>
+                    <field name="active" column_invisible="True"/>
```

Comment:
I'm not sure but aren't fields used in conditions now automatically added?

Reply:
Ho that's neat, I removed them.